src/components/Slider/Slider.tsx

```diff
@@ -76,8 +87,11 @@ const SliderItem = (props: SliderItemProps) => {
                     aria-hidden="true"
                 />
             )}
-            <label
-                htmlFor={isActive ? id : undefined}
+            <div
```

Comment:
Sadly that's also neither the perfect way, but if we don't find anything better, probably it will probably cause fewer problems than the event prevention might.
If I see it right it's not possible to have a `<label>` with the react-aria behavior of the div, is it?

Reply:
Yes that's exactly right. Since it is the default behavior of the label itself that is not playing along nicely with react-aria. Also since the div is not a focusable element but the input is, screen-readers read the aria-label out and ignore the div,  so there's no issue there for now